lib/authlogic/session/cookies.rb

```diff
@@ -96,6 +96,20 @@ def sign_cookie(value = nil)
           rw_config(:sign_cookie, value, false)
         end
         alias_method :sign_cookie=, :sign_cookie
+
+        # Should the cookie be encrypted? If the controller adapter supports it, this is a
+        # measure against cookie tampering and privacy.
```

Comment:
WDYT about "measure to hide the contents of the cookie (e.g. persistence_token)"?

Reply:
updated comment